third_party/libwebp/src/dsp/cpu.c

```diff
@@ -13,6 +13,51 @@
 
 #include "src/dsp/dsp.h"
 
+#if defined(STARBOARD)
+
+#include "starboard/cpu_features.h"
+#include "starboard/log.h"
+#include "starboard/system.h"
+
+bool warn_if_not_enabled(const char * name, bool value) {
```

Comment:
nit: when referring to a pointer I think there should be a space either before or after the asterisk, but not both - go/cstyle#Pointer_and_Reference_Expressions.

Reply:
Well what do you know, i found that we don't run clang-format by default on C files at all, even if i remove third_party excludes. Just did that and it cleaned up rest of spacing below as well.